.github/workflows/FormatCheck.yml

```diff
@@ -42,3 +42,12 @@ jobs:
               write(stdout, out)
               exit(1)
           end'
+
+  format-suggestions:
```

Comment:
With this part below, the part above is mostly obsolete. Could you please remove it?